lib/internal/Magento/Framework/Dto/DtoProcessor/GetHydrationStrategy.php

```diff
@@ -0,0 +1,161 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Framework\Dto\DtoProcessor;
+
+use LogicException;
+use Magento\Framework\Api\SimpleDataObjectConverter;
+use Magento\Framework\Reflection\NameFinder;
+use ReflectionException;
+use Zend\Code\Reflection\ClassReflection;
+
+class GetHydrationStrategy
+{
+    /**
+     * Strategy for setter hydration
+     */
+    public const HYDRATOR_STRATEGY_SETTER = 'setter';
```

Comment:
Should be these constants as public?

Reply:
They are required by other classes like `\Magento\Framework\Dto\DtoProcessor`